frontend/assets/javascripts/src/modules/form/options.js

```diff
@@ -0,0 +1,199 @@
+define(['$', 'bean', 'src/modules/form/validation/display'], function ($, bean, validationDisplay) {
+    'use strict';
+
+    var CURRENCY_ATTR = 'data-currency';
+    var DATA_CURRENCY_SEL = '[' + CURRENCY_ATTR + ']';
+    var BILLING_PERIOD_CONTAINER_EL = '.js-billing-period__container' + DATA_CURRENCY_SEL;
+    var PAYMENT_OPTIONS_CONTAINER_EL = $('.js-payment-options-container')[0];
+    var CARD_DETAILS_NOTE_EL = $('.js-card-details-note')[0];
+    var CARD_NOTE_CURRENCIES_ELS = $(DATA_CURRENCY_SEL, CARD_DETAILS_NOTE_EL);
+    var CARD_NOTE_PERIOD = '.js-card-note-pricing-period';
+    var CARD_NOTE_PRICING_CHARGE = '.js-card-note-pricing-charge';
+    var SUBMIT_BTN_EL = $('.js-submit-input')[0];
+    var DELIVERY_COUNTRY_EL = $('#country-deliveryAddress')[0];
+    var BILLING_COUNTRY_EL = $('#country-billingAddress')[0];
+    var BILLING_ADDRESS_SEL = $('.js-billingAddress-fieldset');
+    var BILLING_ADDRESS_EL = BILLING_ADDRESS_SEL[0];
+    var USE_BILLING_ADDRESS_EL = $('#use-billing-address')[0];
+    var USE_DELIVERY_ADDRESS_EL = $('#use-delivery-address')[0];
+    var BILLING_CTA_SEL = $('.js-toggle-billing-address');
+    var FRIEND_FORM_EL = $('.js-friend-form')[0];
+
+    var checkoutForm = guardian.membership.checkoutForm;
+
+    /**
+     * One way binding for the model object in 'guardian.membership.checkoutForm'.
+     * This object is generated by the backend and mutated as the user interacts with the form.
+     */
+
+    function renderPrices() {
+        hideBillingAddress();
+        selectDeliveryCountry()
+        renderPaymentOptions();
+        renderCardDetailsNotes();
+        renderSubmitButton();
+    }
+
+    var hasParent = function (elem) {
+        return elem.parentNode;
+    }
+
+    function hideBillingAddress() {
+        if (!hasParent(BILLING_ADDRESS_EL) && checkoutForm.showBillingAddress) {
```

Comment:
in fact is there a strong reason why you need to add & remove from the DOM here rather than just hiding/showing the billing address? if not I think hiding/showing would be simpler